cloud_images/centos7/install_prereqs.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+set -o errexit -o nounset -o pipefail
+
+echo ">>> Kernel: $(uname -r)"
+echo ">>> Updating system"
+yum update --assumeyes
+
+echo ">>> Disabling SELinux"
```

Comment:
Need a followup jira to make this and the SSH / Web installer use the same bash to setup pre-requisites. Don't want lots of slightly non-matching implementations around.